logging/Limiting_Access_to_Logs.md

```diff
@@ -15,39 +17,86 @@ only for a specified namespace.
-This access is controlled by a combination of Kibana tenant spaces and 
+This access is controlled by a combination of Kibana-tenant spaces and 
 Open Distro for Elasticsearch roles, back-end roles, and role mappings. 
 Scripts are provided to create everything that is needed for each 
-scenario.
+scenario. For details, see [Implement Access Controls for a Tenant or Namespace](Limiting_Access_to_Logs.md#implement_access).
+
+## <a name="logadm"></a>The Logadm User and Its Access Controls
+
+### Overview
+
+As of the 1.1.3 release, the following items are created during the deployment process:
+
+- a set of role-based access controls (RBACs)
+- the logadm user, which is linked to the new RBACs
+
+You can disable the creation of the logadm user during the deployment process. Set the LOG_CREATE_LOGADM_USER environment variable to 'false'. The default is to create the logadm user.
+  
+### Logadm Access Controls
 
-## Implement Access Controls for a Tenant or Namespace
+The logadm user is the day-to-day user for log monitoring for cluster administrators. The logadm user has access to all of the log messages collected by the log-monitoring components. This means that the logadm user has access to the following messages: 
+
+- log messages from all SAS Viya namespaces and all tenants within those namespaces.
+- log messages from all non-SAS Viya namespaces including the Kubernetes infrastructure namespaces.
+ 
+Although the logadm user can access all of the log messages, logadm can access only the "cluster_admins" Kibana-tenant space.
+
+The access controls are designed to provide logadm users with full cluster access to perform the log-monitoring activities needed for their job.  Limiting these users to the "cluster_admins" Kibana-tenant space helps in the following ways:
+
+- simplifies the Kibana user experience.
+- eliminates the potential confusion that could otherwise occur if the user could  access Kibana content in the other Kibana-tenant spaces.
+
+
+| Back-end Role | Role | Purpose |
+| --- | --- | --- |
+|     | v4m_kibana_user | Grants user access to the Kibana functionality needed. (This role is not unique to the logadm user). |
+|     | search_index | Grants access to all collected log messages. |
+|     | tenant_cluster_admins | Grants access to the cluster_admins Kibana-tenant space.|
+|V4MCLUSTER_ADMIN_kibana_users |    | Provides an easy way to link the above roles with a user.
+
+Linking users to the back-end role of V4MCLUSTER_ADMIN_kibana_users via the Kibana security plug-in grants those users the access describe above.
+
+You can create new users and link them to these access controls by running the user.sh script.  For example, to create a user called "Chen" and link her to these access controls, submit the following command:
+
+<pre>
+/logging/bin/user.sh CREATE --user <i>username</i> --password <i>password</i> --namespace _all_ --tenant _all_
+</pre>
+
+Where: 
+
+- *username* is chen.
+- *password* is the password to WHAT?.
```

Comment:
Move this portion (starting at line 60) of this section (showing the creation of the user "chen") to the "Managing User Accounts for Restricted Access" section.  Add it after the general discussion of how to create a user as a "special case".

Reply:
I'll make the move and see how it flows. Thanks!